### DISCUSSION TITLE
Is 36.62mh/s Hash slow on ETH with NVIDIA GeForce RTX 2070 SUPER ?

### DISCUSSION THREAD

**Discussion by SSBasra** [Lolliedieb/lolMiner-releases#450] [Q&A]
I am new to Mining so not sure what to expect

I am running a Intel i9 9900K @ 5GHz
NVIDIA GeForce RTX 2070 SUPER @ factory settings
GibaByte Z390 Designare-CF 
Corsair 2x32Gb (64Gb) DDR4 Memory
Karken X3 Liquid cooling
Storage AData SX8200PNP (2TB)

I am running lolminer v1.26 with standard eth batch file
lolMiner.exe --algo ETHASH --pool !POOL! --user supersinghxx !WALLET! --watchdog exit

What am I doing wrong if anything?  I have a fast CPU and I think a good video card.  I was expecting more use of both.  If I run multiple instances of the mine_eth.bat will it improve or have I got the wrong idea.  Do I need to configure my graphics card somehow or just modify the batch file so it uses my resources better?

I am a newbie to mining but want to learn.

and getting these figures from my machine 

Setup Miner...
OpenCL driver detected.
Number of OpenCL supported GPUs: 1
Cuda driver detected.
Number of Cuda supported GPUs: 1
Device 0:
    Name:    GeForce RTX 2070 SUPER
    Address: 2:0
    Vendor:  NVIDIA Corporation
    Drivers: Cuda, OpenCL
    Memory:  8192 MByte
    Active:  true (Selected Algorithm: ETHash Cuda)

my GPU is running @1% load and cpu @ 4%
my temp is 60c and cpu is 39C

---------------------------------------------
Statistics (22:52:17); Mining: Ethash
Connected to: eth-eu.f2pool.com:6688
Uptime: 0h 2m 30s
                       GPU 0   Total
Speed (mh/s):          34.02   34.02
Pool (mh/s):           57.27   57.27
Best Share:            21.6G   21.6G
Shares (A/S/Hw):       1/0/0   1/0/0
Power (W):             175.7   175.7
Efficiency (mh/s/W):   0.193   0.194
Temp (deg C):             60
Fan Speed (%):            42
---------------------------------------------
Average speed (30s): 36.62 mh/s

**Comment by MiroslauF** [comment]
Mining has nothing to do with CPU in this configuration.
Hard to believe that your GPU is running at 1% load when you see 34mh/s Speed.
34mh/s is good for 2070 SUPER. :)

**Reply by SSBasra** [reply]
Thanks for the feedback, yes I was suprised.  I also installed Nice hash, and this put the GPU usage up to 100%, I suspect that NZCTCam was telling me porky pies.

**Comment by professorNoob** [comment]
You'll want to change your GPU settings. Currently you are using a lot of power for not a lot of hashrate. Download MSI Afterburner and modify your setup. I have a 2070. 
My settings are
Power Limit: 70%
Core clock: -50
Memory clock: +1000 (7800Mhz)

I use 130W and get between 41 and 43mh/s.

You can google what settings other people are using if you don't want to experiment.

**Reply by SSBasra** [reply]
I have downloaded MSI afterburner and will try once my AVG antivirus issue preventing me running are resolved

**Comment by Sinancetintas** [comment]
Same as me, I have RTX 2060 Super.
GPU usage: %1

Speed (mh/s):            35.29
Pool (mh/s):             34.52
Best Share:               2.1T
Shares (A/S/Hw):       189/2/0
Power (W):               158.8
Efficiency (mh/s/W):     0.222
Temp (deg C):               59
Fan Speed (%):              49

**Comment by Gibswal** [comment]
These are my results below.
Speed (mh/s): 62.10
Pool (mh/s): 59.98 
Best Share: 6.9T
Shares (A/S/Hw): 150/0/0
Power (W): 158.4
Efficiency (mh/s/W): 0.393
Temp (deg C): 54
Fan Speed (%): 55

Granted I am using a 2080ti so will get better results, I would suggest though putting your memory clock to about +1000 (if you can get it higher without any errors just keep going up in small increments) me personally I use +1337 on memory clock and never get any errors but any higher I get some occasionally.
My power limit is set to 55%
Core clock is -200
You say you're using default settings so download an OC tool like MSI afterburner.
You want to get your efficiency as high as possible without losing speed of the mining.
Me personally I have the same CPU I have it clocked at 4.4GHz and the CPU to my knowledge won't affect your speed in any way.

My friend with the same card suggested these settings. He is getting around 40 mh/s.
Core Voltage (VDD): N/A
Power Limit (%): ~72% or 150 (W)
Core Clock (CClock): -100
Memory Clock (MClock): +900
Fan Speed (%): 70% ± based on your temps.

I know its a bit late of an answer so I assume you have probably gotten the hang of it by now but I hope this helps somewhat.

**Reply by SSBasra** [reply]
Thanks I havent resolved my issue yet as AVG antivirus has deceided not to play ball and wont let me put the nice hash / lol miner as a exception. so if anyone can recommend an good aternate free antivirus that it will work with that would be great

**Reply by Gibswal** [reply]
I use malwarebytes

**Comment by SSBasra** [comment]
Thanks Gibswal & ProfessorNoob, I used your and others suggestions downloaded MSI afterburner, tweaked my settings to

![image](https://user-images.githubusercontent.com/34506551/116594748-77dabd80-a91a-11eb-9dbb-8163a3325245.png)

and now getting 41.81 - 43.5 MH/s 167W with both lol and t-rex miners

![image](https://user-images.githubusercontent.com/34506551/116595020-c5efc100-a91a-11eb-8555-74a36d589a70.png)

Next job maybe build / buy a dedicated rig once I see my ROI

**Comment by Stolen1st** [comment]
I'm new to all this stuff and late to this post.  I also run a 2070 SS on and i7 AWR11.

Getting about 40.40mh/s with MSI at the current settings from above @SSBasra.
Core Clock -200
Mem Clock +900
Power Limit 72%
Temp limit 71*c
fan speed set to Auto, running at 62% at 62*c

Mem is running at 7700 Mhz
GPU 1680 Mhz

Curious as to everyone else's setup with the 2070 SS, finding this post definetly helped me get a better understanding of what I'm some what doing... I think... :/

I also use LOL Miner 1.28a.. is there is difference besides looks on either LOLminer or T-Rex, EasyMiner?

Do anyone Have an Alienware machine?
if so, do you use MSi or AW Command center to OC/tweak performance?

**Reply by professorNoob** [reply]
So, the hash rate is dependent mostly on the gpu memory speed. Your cpu doesn't play any part in ethereum mining. You can increase or decrease the memory clock to see changes in hash rate. Once you get a stable overclock (no invalid shares) you can adjust the power limit lower to see how low you can set it without your hash rate going down.

Since mining is memory intensive, it's actually the memory temps that are more important. Not all cards have sensors for that, but I've found it helpful to look at the "Hotspot" or "Junction" temperature on a tool like He-Monitor. Under 90 degrees is what you should aim for

**Reply by Gibswal** [reply]
> I'm new to all this stuff and late to this post. I also run a 2070 SS on and i7 AWR11.
> 
> Getting about 40.40mh/s with MSI at the current settings from above @SSBasra.
> Core Clock -200
> Mem Clock +900
> Power Limit 72%
> Temp limit 71_c fan speed set to Auto, running at 62% at 62_c
> 
> Mem is running at 7700 Mhz
> GPU 1680 Mhz
> 
> Curious as to everyone else's setup with the 2070 SS, finding this post definetly helped me get a better understanding of what I'm some what doing... I think... :/
> 
> I also use LOL Miner 1.28a.. is there is difference besides looks on either LOLminer or T-Rex, EasyMiner?
> 
> Do anyone Have an Alienware machine?
> if so, do you use MSi or AW Command center to OC/tweak performance?

Try +1000 memory, Power limit down to like 55-60%

**Reply by Stolen1st** [reply]
That was totally helpful @Gibswal, HWMonitor is what I wanted.. Something to monitor the system a bit in-depth. hitting 41.05 mh/s pretty steady with that change up @ Power limit to 55-60% (60 atm). 

the knowledge helped also, @ Memory and reducing power limit without loss of mh/s. If I got this mining $100.00 a month, I'd be pretty impressed... I love stuff like this.. the money is fun, but pushing the system to peak performance and longevity is paramount.  I see a lot of people jumping on these more high end cards. Why not push smaller cards, granted the number of cards required would be greater.  But overall cost and maintenance vs ROI vs TIME vs efficiency vs availability...

I'm sure someone's thought of all this...

**Reply by Gibswal** [reply]
I'm rocking a 2080ti I get around 61 MH/s so you ain't doing too bad yourself. I've managed to nab myself a 3090 which is on its way which should triple my output. It is mainly for gaming though but if I can hit $400 a month it's free money practically I can't say no to that, Like if you do any gaming at all and don't want to hurt your hashtate so much just try using Rivatuner it comes with MSI afterburner, I limit my games to 30fps lol but I still get like 50MH/s I keep the settings the same for gaming as I would for mining. Happy mining!

**Reply by Stolen1st** [reply]
Yea, I was about to tackle that next @ Gaming and mining.. I'll definitely check that out. I bought this for gaming, but I'm all about a Buck and no money is better than residual income. lol

Appreciate the help, I'm on platforms hit me up..
